examples/openvino/stable-diffusion-quantization/quantize.py

```diff
@@ -0,0 +1,1030 @@
+                # (this is the forward diffusion process)
+                noisy_latents = noise_scheduler.add_noise(latents, noise, timesteps)
+
+                # Get the text embedding for conditioning
+                encoder_hidden_states = text_encoder(batch["input_ids"])[0]
+
+                # Predict the noise residual and compute loss
+                noise_pred = unet(noisy_latents, timesteps, encoder_hidden_states).sample
+                loss = F.mse_loss(noise_pred.float(), noise.float(), reduction="mean")
+
+                # Gather the losses across all processes for logging (if we use distributed training).
+                avg_loss = accelerator.gather(loss.repeat(args.train_batch_size)).mean()
+                train_loss += avg_loss.item() / args.gradient_accumulation_steps
+
+                compression_loss_unet = compression_controller.loss()
+                loss = loss + compression_loss_unet
+
+                # Backpropagate
+                accelerator.backward(loss)
+                if accelerator.sync_gradients:
+                    accelerator.clip_grad_norm_(unet.parameters(), args.max_grad_norm)
+                optimizer.step()
+                lr_scheduler.step()
+                optimizer.zero_grad()
+
+            # Checks if the accelerator has performed an optimization step behind the scenes
+            if accelerator.sync_gradients:
+                if args.ema_device:
+                    ema_unet.step(orig_unet.parameters())
+                progress_bar.update(1)
+                global_step += 1
+                accelerator.log({"train_loss": train_loss}, step=global_step)
+                train_loss = 0.0
+
+            logs = {"step_loss": loss.detach().item(), "lr": lr_scheduler.get_last_lr()[0]}
+            progress_bar.set_postfix(**logs)
+
+            if global_step >= args.max_train_steps:
+                break
+
+    # Create the pipeline using the trained modules and save it.
+    accelerator.wait_for_everyone()
+    if accelerator.is_main_process:
+        unet = accelerator.unwrap_model(unet)
+        if args.ema_device:
+            ema_unet.copy_to(orig_unet.parameters())
+        
+        if args.push_to_hub:
+            repo.push_to_hub(commit_message="End of training", blocking=False, auto_lfs_prune=True)
+
+    accelerator.end_training()
+    
+    # Export optimized pipline to OpenVINO
+    export_unet = compression_controller.strip(do_copy=False)
+    export_pipeline = StableDiffusionPipeline(
+        text_encoder=text_encoder,
+        vae=vae,
+        unet = export_unet,
+        tokenizer=tokenizer,
+        scheduler=noise_scheduler,
+        safety_checker=pipeline.safety_checker,
+        feature_extractor=pipeline.feature_extractor,
+    )
+
+    with tempfile.TemporaryDirectory() as tmpdirname:
+        export_to_onnx(export_pipeline, tmpdirname)
+        export_to_openvino(export_pipeline, tmpdirname, Path(args.output_dir) / "openvino")
```

Comment:
It would be nice to have an export function in optimum directly for this kind of case, will open a PR for this